src/Sulu/Component/SmartContent/ContentType.php

```diff
@@ -203,6 +203,7 @@ public function getDefaultParams(PropertyInterface $property = null)
                 'presentAs' => $configuration->hasPresentAs(),
             ],
             'datasource' => $configuration->getDatasource(),
+            'deep-link' => new PropertyParameter('deep-link', $configuration->getDeepLink()),
```

Comment:
Actually this line is connected to the comment in the twig template above. When doing a quick search on `PropertyParameter` I find examples written with underscores, dashes and in camel case. We should agree on one (and it should not be the dash, since it is unnecessarily complicated to handle in twig).